playbooks/installer/ospd/overcloud/run.yml

```diff
@@ -20,6 +20,7 @@
   roles:
       - {role: "installer/ospd/overcloud/storage/ceph/", when: installer.overcloud.storage.backend is defined and installer.overcloud.storage.backend == "ceph"}
       - {role: "installer/ospd/overcloud/storage/netapp/", when: installer.overcloud.storage.backend is defined and installer.overcloud.storage.backend == "netapp"}
+      - {role: "installer/ospd/loadbalancer/", when: provisioner.topology.nodes.loadbalancer is defined }
```

Comment:
please change: when: provisioner.topology.nodes.loadbalancer is defined
to: when: "'loadbalancer' in groups"